packages/react-app/src/lib/types.ts

```diff
@@ -14,6 +14,8 @@ type Giveaway = {
   isInvalid: boolean;
   rules?: string;
   requirements?: Requirements;
+  winningChance: number;
+  status: string;
```

Comment:
I was checking the giveaway type and it does not contain a `status` property, instead I think we should use `isInvalid` property which is already defined in this type

Reply:
We can keep the `winningChance` to avoid having errors on the `GiveawayMainContent` component. Could you also remove this part from the GiveawayCard please?

```
        {giveaway.status === GiveawayStatus.INVALID && (
          <div className="invalid-giveaway">
            <div className="center-text">
              <img className="invalid-icon" src={SadEmoji} alt="SadEmoji" />
              <Typography className="invalid-message">
                Not enough participants
              </Typography>
            </div>
          </div>
        )}
```

which seems to be duplicated of:

```
        {giveaway.isInvalid && (
          <div className="invalid">
            <div style={{ textAlign: 'center' }}>
              <img className="icon" src={SadEmoji} alt="SadEmoji" />
              <Typography className="message">Not enough participants</Typography>
            </div>
          </div>
        )}
```